src/color/brand.json

```diff
@@ -69,13 +69,18 @@
           "wcag": "n/a"
         },
         "400": {
-          "value": "0074ca",
+          "value": "0074cb",
           "comment": "{comments.color.brand.value.comment}",
           "public": true,
           "neutral": true,
           "usage": "Transactional colors to move guests through task-based interactions.",
           "wcag": "n/a"
         },
+        "400-opacity-20": {
```

Comment:
What uses this token?

Reply:
[color-ui-disabled-onLight](https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/ui.json#L52)